.github/workflows/build-python.yml

```diff
@@ -209,4 +209,11 @@ jobs:
           poetry new ./install-test
           cd ./install-test
           sed -i 's/^\(python *= *\).*$/\1">=3.10,<3.12"/' pyproject.toml
-          poetry add "$(ls ../dist/*.whl)[server]"
\ No newline at end of file
+          poetry add "$(ls ../dist/*.whl)[server]"
```

Comment:
why are we still using poetry if there's PDM?

Reply:
Test before was with poetry, and habits die hard ^^'
Can change, have to see how to create empty project with pdm

Reply:
I'd actually only use pdm to create a virtualenv (since pdm is already installed anyway)
`pdm venv create XXX`

and then activate it and install giskard with regular `pip`

```
pip install giskard
python -c 'import giskard'
```

Reply:
this makes it way more readable too

Reply:
and closer to what people will do